### PR TITLE
Indexing Trec Core (#187)

### PR DESCRIPTION
**Pull request by rosequ**
- indexer binding for TREC Core collection
- parsing NytCorpus documents (provided by the organizers and is Apache licensed)
The parsing code is _verbatim_ now and needs to be slightly modified. This is an initial PR, so that we can track back to the original files.

### CODE REVIEW

File: docs/experiments-core17.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Anserini Experiments on Gov2
```
Comment (lintool) [inline docs/experiments-core17.md]: This still says Gov2... change to NYT?

Comment (rosequ) [inline docs/experiments-core17.md]: Oh sorry, forgot to change it.

Comment (rosequ) [inline docs/experiments-core17.md]: Shall I go ahead and make the changes to the original file and make another PR?

Comment (lintool) [inline docs/experiments-core17.md]: Fix the issue, push again, and this PR will get updated automatically.
